tests/ttnn/unit_tests/test_squeeze.py

```diff
@@ -0,0 +1,27 @@
+# SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+
+# SPDX-License-Identifier: Apache-2.0
+
+import pytest
+
+import torch
+
+import ttnn
+
+
+@pytest.mark.parametrize(
+    "input_shape, dim, output_shape",
+    [
+        # ((1,256,1), -1, (1,256)),
+        ((1, 1, 1, 256), 2, (1, 1, 256)),
```

Comment:
Torch supports `dim` argument to specify a dimension or a list of dimensions. Can we reflect that in a test and indicate that only 0 is supported if thats the case?